compiler/src/dmd/common/README.md

```diff
@@ -5,4 +5,4 @@
 | [bitfields.d](https://github.com/dlang/dmd/blob/master/compiler/src/dmd/common/bitfields.d) | Pack multiple boolean fields into bit fields                    |
 | [file.d](https://github.com/dlang/dmd/blob/master/compiler/src/dmd/common/file.d)           | Functions and objects dedicated to file I/O and management      |
 | [outbuffer.d](https://github.com/dlang/dmd/blob/master/compiler/src/dmd/common/outbuffer.d) | An expandable buffer in which you can write text or binary data |
-| [string.d](https://github.com/dlang/dmd/blob/master/compiler/src/dmd/common/string.d)       | Common string functions including filename manipulation         |
```

Comment:
The description should be updated when those functions are moved out

Reply:
I agree